File: Estrutura_Repetição/C/Ex1060/Main.c
#include <stdio.h>

int main() {
    int positivos = 0;
    float valor = 0;
    for (int i = 0; i <= 5; i++) {
        scanf("%f", &valor);
        if (valor > 0) {
            positivos += 1;
        }
    }
    printf("%d valores positivos\n", positivos);

    return 0;
}

File: Estrutura_Repetição/C#/Ex1060/Main.cs
using System;

class MainClass {
    static void Main(string[] args) {
        int positivos = 0;
        float valor = 0;

        for (int i = 0; i <= 5; i++) {
            valor = float.Parse(Console.ReadLine());
            if (valor > 0) {
                positivos++;
            }
        }
        Console.WriteLine($"{positivos} valores positivos");
    }
}